primitives/arithmetic/src/fixed_point.rs

```diff
@@ -928,14 +931,14 @@ macro_rules! implement_fixed {
 			}
 		}
 
-		#[cfg(feature = "std")]
+		#[cfg(feature = "serde")]
 		impl sp_std::fmt::Display for $name {
```

Comment:
Necessary because Serialize uses `to_string()`.

Reply:
AFAIK we shouldn't need this `#[cfg(feature)]` gate as `Display` is in `core`, no?

Reply:
Personally I'd remove the feature gate. This one extra impl won't realistically affect anything even if it's not actually used, and it's just simpler to have this always available.

Reply:
Removed in https://github.com/paritytech/substrate/pull/13027/commits/09f6049e998fa76b9d1343b5294938b181872379